mycore-orcid2/src/main/resources/xsl/orcid2/v3/work2mcr_generic.xsl

```diff
@@ -208,11 +208,13 @@
   <xsl:template match="common:last-modified-date" />
 
   <xsl:template match="common:language-code">
-    <mods:language>
-      <mods:languageTerm authority="rfc5646" type="code">
-        <xsl:value-of select="document(concat('language:',text()))/language/@xmlCode" />
-      </mods:languageTerm>
-    </mods:language>
+    <mods:recordInfo>
+      <mods:languageOfCataloging>
+        <mods:languageTerm authority="rfc5646" type="code">
+          <xsl:value-of select="document(concat('language:',text()))/language/@xmlCode" />
```

Comment:
Should check if a corresponding language classification value exists and handle the case when this is not the case.